_data/form.yml

```diff
@@ -0,0 +1,831 @@
+    label: |
+
+      You need to apply for separate permits for any:
+
+      - Additions or changes to auto runways
+
+      - Building extensions
+
+      - Changes of occupancy
+
+      - Electrical work
+
+      - Plumbing work
+
+      - Street space used during construction
+
+      Even if we approve this application, the status of these permits will not change.
+
+  - type: header
+    label: Your identity
+    level: 4
+
+  - type: paragraph
+    label: |
+      By checking the box below, you certify that:
+
+      - You are either the owner, the lessee, or an authorized agent for them
+
+      - You are authorized to sign this permit application
+
+  - label: I declare under penalty of perjury that my application is true and correct.
+    type: checkbox
+
+  - label: I hereby certify and agree that if I am issued a permit for the construction described in this application, all the provisions of the permit and all laws and ordinances thereto will be complied with.
+    type: checkbox
+
+- title: Terms
+  fields:
+  - label: Hold Harmless Clause
+    type: header
+    level: 4
+
+  - label: |
+      The permittee(s) by acceptance of the permit, agree(s) to indemnify and hold harmless the City and County of San Francisco from and against any and all claims, demands and actions for damages resulting from operations under this permit, regardless of negligence of the City and County of San Francisco, and to assume the defense of the City and County of San Francisco against all such claims, demands or actions.
+
+      I hereby affirm under penalty of perjury the following declaration(s):
+    type: paragraph
+
+  - group: comp-I
+    fields:
+    - label: >
+        I have a certificate of consent to self-insure for workers' compensation. I will maintain the certificate for the work described in this permit.
+      type: checkbox
+
+  - group: comp-II
+    fields:
+    - label: >
+       I have workers' compensation insurance, as required by Section 3700 of the Labor Code. I will maintain the insurance for the work described in this permit.
+      type: checkbox
+
+  - group: comp-III
+    fields:
+    - label: The work will cost $100 or less.
+      type: checkbox
+
+  - group: comp-IV
+    fields:
+    - label: |
+       The California worker's compensation laws do not apply to me. When I perform the work described in this permit, I do not plan to employ anyone in a way that would subject me to them.
+
+        If California worker's compensation laws do apply to this project, and I fail to comply with them, I understand that the city will revoke my permit.
+      type: checkbox
+
+  - group: comp-V
+    fields:
+    - label: |
+       I certify I will hire a contractor who complies with California worker's compensation laws for the work described in this permit. Before starting any work, they will file a finished copy of this form with the Central Permit Bureau.
+      type: checkbox
+
+- title: Disclosures for Owner-Builders
```

Comment:
Persis: try to get 15 minutes with the City Attorney and see whether there's anything we can remove

Reply:
Scott: This is a small group, but they would need the most help to get through this form